media/img/icons/cryptomining-pink.svg

```diff
@@ -0,0 +1 @@
+<svg width="60" height="50" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="M46.005 7h-24a6.009 6.009 0 00-4.8 2.4l-3.6 4.8c-1.482 1.98-.072 4.8 2.4 4.8h30V7z" id="a"/></defs><g fill="none" fill-rule="evenodd"><path d="M34 7V5.5A1.5 1.5 0 0032.5 4h-3A1.5 1.5 0 0028 5.5v39a1.5 1.5 0 001.5 1.5h3a1.5 1.5 0 001.5-1.5V7z" fill="#FF8AC5"/><use fill="#E31587" xlink:href="#a"/></g></svg>
```

Comment:
We usually include both width/height and viewBox in SVGs. I can't remember which browser needs which... I know old IEs didn't support viewBox, and maybe they ALL support width/height so this could be fine as-is. But it still seems like it can't hurt to have both. So here we can add `viewBox="0 0 60 50"` (and update the others as well).

Reply:
Could I just turn around users who are on a different browser? Send them to the homepage or something.

Reply:
Adding viewBox is certainly not a blocker, just a nice-to-have (and I'm pretty sure as long as width and height are there viewBox is optional anyway... it might be different if it was only viewBox and no width/height). I don't think we need to forcibly prevent non-Firefoxes from loading these pages, they're very unlikely to get any odd traffic.